server/schemas/orders.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+let ordersSchema = new mongoose.Schema({
+  user: { type: mongoose.Schema.Types.ObjectId, ref: "users" }, // if no user need to validate this.
```

Comment:
For database optimization, you could index this field

Reply:
index the user.